### PR TITLE
Add edge case testing for adding multiple pets from same app

### PR DESCRIPTION
**Pull request by brian-greeson**
- Added expects to test approving multiple pets on one application
- [x] All tests passing?
- Fixes: #24

### CODE REVIEW

File: spec/features/pet_applications/show_spec.rb [hunk 1]
```diff
@@ -79,5 +88,15 @@
...
+
+    within "section.application-#{app_1.id}-pets" do
+      find("#approve-#{pet_2.id}").click
+    end
+
+    expect(current_path).to eq("/pets/#{pet_2.id}")
+    expect(page).to have_content("pending")
+    expect(page).to have_content("On hold for #{app_1.name}")
   end
 end
```
Comment (SMJ289) [inline spec/features/pet_applications/show_spec.rb]: Awesome, we already had the functionality and just needed a test! 💯

Comment (brian-greeson) [reply spec/features/pet_applications/show_spec.rb]: @SMJ289
